internal/blueprint/blueprint.go

```diff
@@ -74,6 +76,12 @@ func (b *Blueprint) Initialize() error {
 	if err != nil {
 		return fmt.Errorf("Invalid 'version', must use Semantic Versioning: %s", err.Error())
 	}
+
+	err = b.CryptPasswords()
```

Comment:
This works because `Initialise()` is always called after serialisation into the struct?

Reply:
Correct.